user/firejail/template.py

```diff
@@ -0,0 +1,23 @@
+pkgname = "firejail"
+pkgver = "0.9.72_git20250221"
+pkgrel = 0
+_commit = "733f9a9c487358ba73b06186ed348f086d8ac839"
```

Comment:
we don't package git software

Reply:
The latest release/tag dates from 2023 but the repo is active. Please advise.

Reply:
In the mean while, new versions hev been tagged with versions number by the author. There is no need anymore to select a non-version-tagged commit.